src/primitives/block.cpp

```diff
@@ -25,9 +25,8 @@ std::string CBlock::ToString() const
         hashMerkleRoot.ToString(),
         nTime, nBits, nNonce,
         vtx.size());
-    for (unsigned int i = 0; i < vtx.size(); i++)
-    {
-        s << "  " << vtx[i]->ToString() << "\n";
+    for (const auto &tx : vtx) {
```

Comment:
Hm, when I look at your last patch #10916 you actually do the same.

> first argument is a reference type

I would call that an unnamed reference to type std::string :trollface:

I guess it's not that bad for arguments and return types though, I'll look into it.